azavea/ansible-cartodb
# Provision foreign data wrapper setup

Updates the provisioner to properly setup the FDW configuration when using the `cartodb-org` fork of cartodb, when using the `ansible/group_vars/all.cartodb-org.example` group vars file instead of `all.example`. 
### To Test

Clone the cartodb-org fork to `../cartodb` and checkout the `bbg_production` branch. Then copy the `ansible/group_vars/all.cartodb-org.example` file to `ansible/group_vars/all`. Then run `vagrant up` and wait for the provisioner. Once it runs, you should be able to properly import FDW synced tables via the data library in the dashboard.

The existing `all.example` file will still work for the current master branch of hte upstream cartodb repository, and skips all of the local common data/FDW specific tasks.

I'm reprovisioning one more time on the `bbg_production` branch to verify this works as expected.
